_pytest/python_api.py

```diff
@@ -584,9 +585,10 @@ def raises(expected_exception, *args, **kwargs):
 
     """
     __tracebackhide__ = True
-    for exc in filterfalse(isclass, always_iterable(expected_exception)):
+    base_type = (type, text_type, binary_type)
+    for exc in filterfalse(isclass, always_iterable(expected_exception, base_type)):
         msg = ("exceptions must be old-style classes or"
-               " derived from BaseException, not %s")
+                " derived from BaseException, not %s")
```

Comment:
this one is the unfortunate dedent mistake that broke the linting job ^^

Reply:
oh man. thanks for the pointer